packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -17,4 +17,20 @@ describe("<DateRangeInput>", () => {
         const component = mount(<DateRangeInput />);
         expect(component.find(InputGroup).length).to.equal(2);
     });
+
+    it("startInputProps.inputRef receives reference to HTML input element", () => {
+        const inputRef = sinon.spy();
+        // full DOM rendering here so the ref handler is invoked
+        mount(<DateRangeInput startInputProps={{ inputRef }} />);
+        expect(inputRef.calledOnce).to.be.true;
+        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
```

Comment:
honestly this doesn't feel necessary. you're testing incredibly basic logic that can't really break. it's basically that we have that one line of code `safeInvoke(inputRef)` exists.

i'm not opposed to not bothering with these tests at all.